src/types_and_consts_additions.jl

```diff
@@ -46,23 +46,29 @@ function Base.convert(::Type{SparseMatrixCSC}, J::SUNMatrix)
     m = mat.M
     n = mat.N
     nzval = unsafe_wrap(Array,mat.data, (mat.NNZ), own=false)
-    SparseMatrixCSC(m,n,colptr,rowval,nzval)
+    SparseArrays.SparseMatrixCSC(m,n,colptr,rowval,nzval)
 end
 
 abstract type SundialsMatrix end
 struct DenseMatrix <: SundialsMatrix end
 struct BandMatrix <: SundialsMatrix end
 struct SparseMatrix <: SundialsMatrix end
 
-abstract type SundialsLinSol end
-struct Dense <: SundialsLinSol end
-struct Band <: SundialsLinSol end
-struct SPGMR <: SundialsLinSol end
-struct SPFGMR <: SundialsLinSol end
-struct SPBCGS <: SundialsLinSol end
-struct PCG <: SundialsLinSol end
-struct PTFQMR <: SundialsLinSol end
-struct KLU <: SundialsLinSol end
+abstract type SundialsLinearSolver end
+struct Dense <: SundialsLinearSolver end
+struct Band <: SundialsLinearSolver end
+struct SPGMR <: SundialsLinearSolver end
+struct SPFGMR <: SundialsLinearSolver end
+struct SPBCGS <: SundialsLinearSolver end
+struct PCG <: SundialsLinearSolver end
+struct PTFQMR <: SundialsLinearSolver end
+struct KLU <: SundialsLinearSolver end
+struct LapackBand <: SundialsLinearSolver end
+struct LapackDense <: SundialsLinearSolver end
```

Comment:
we might want to default to the Lapack versions. Also, let's make sure the docs get updated to add these.

Reply:
Is this an action item for me?

Reply:
Action item for @YingboMa to be aware of.